tests/integration-tests/clusters_factory.py

```diff
@@ -235,13 +235,15 @@ def get_cluster_instance_ids(self, node_type=None, queue_name=None):
 
     def export_logs(self, bucket, output, bucket_prefix=None):
         """Run pcluster export-cluster-logs and return the result."""
-        cmd_args = ["pcluster", "export-cluster-logs", self.name, "--bucket", bucket, "--output", output]
+        cmd_args = ["pcluster", "export-cluster-logs", "--cluster-name", self.name, "--bucket", bucket, "--output",
```

Comment:
let's align the other commands too: list-cluster-logs get-cluster-log-events etc, let's double check all of them.